spec/spec_helper.rb

```diff
@@ -121,9 +121,10 @@
 
       # Allow Capybara to make localhost requests and also contact the
       # google api chromedriver store
+      # add googlechromelabs.github.io and edgedl.me.gvt1.com to work with Chrome v116+
       WebMock.disable_net_connect!(
         allow_localhost: true,
-        allow: %w[chromedriver.storage.googleapis.com]
+        allow: %w[chromedriver.storage.googleapis.com googlechromelabs.github.io edgedl.me.gvt1.com]
```

Comment:
Will we need to revert this change once we upgrade to ruby 3?

Reply:
When we upgrade to Ruby 3, I'm thinking we should upgrade to Selenium 4.11+.

According to https://github.com/titusfortner/webdrivers:
"If you can update to the latest version of Selenium (4.11+), please do so and stop requiring this gem."

In that case, we'd remove the original `chromedriver.storage.googleapis.com`. But I'm not sure if Selenium will need `googlechromelabs.github.io` and `edgedl.me.gvt1.com`.

Reply:
Sounds great!

@aaronskiba please add a ticket so we don't forget to follow the suggestions after the upgrade to ruby 3.